### ISSUE TITLE
Which files are generated?

### ISSUE THREAD

**Issue by JustinBeckwith** [googleapis/nodejs-logging#110]
👋 hey folks!  Just curious - which files in this repo are auto-generated? Which ones shouldn't I change?

**Comment by stephenplusplus** [comment]
Sorry for the lag here! `src/v*` is completely generated. `src/index.js` is partially generated, such as the imports and the contstructor... but I haven't actually run the generator as of yet, so I'm not exactly sure if this is a half-robot/half-human process, or what. (@alexander-fenster?)

**Comment by alexander-fenster** [comment]
I replied to @JustinBeckwith  in hangouts and did not see this issue. I replied with pretty much the same text as you @stephenplusplus :)

Normally, everything under `src/v*` is completely generated, as for `src/index.js`, it's auto-generated in most cases, the notable exception is when we need to support two different versions at once, OR when the project has manual code (e.g. spanner or pubsub). @JustinBeckwith has created https://github.com/googleapis/gapic-generator/issues/1986 which will make it easier to tell if the file is autogen'd or not (I wonder why it was not done before :) )
